source/hardware/controllers/roland_dj_505.rst

```diff
@@ -114,36 +114,37 @@ Deck Section (p. 5-6)
-12        :hwlabel:`KEY LOCK`                                              Toggle key lock.
-12        :hwlabel:`SHIFT` + :hwlabel:`KEY LOCK`                           Cycles through tempo slider range (8%, 16%, 50%).
-13        Tempo slider                                                     Adjust song playback speed (and pitch if key lock if off).
-========  ===============================================================  ==========================================
+========  =====================================================================  ==========================================
+No.       Control                                                                Function
+========  =====================================================================  ==========================================
+1         Jog Wheel (top surface)                                                Perform scratch operation.
+1         Jog Wheel (outer edge)                                                 Rotate to lower/raise playback speed and pitch if key lock is off).
+1         :hwlabel:`SHIFT` + Jog Wheel (top surface)                             Search fast through the track location.
+2         :hwlabel:`SLIP` button                                                 Hold to turn on slip mode temporarily or double press to turn it on permanently.
+2         :hwlabel:`SHIFT` + :hwlabel:`SLIP` button                              Toggle vinyl control mode.
+3         :hwlabel:`SHIFT` button                                                Hold down to access other functions.
+4         :hwlabel:`SYNC` button                                                 Match tempo and phase of other deck. Long press to set Deck as Sync Leader.
+4         :hwlabel:`SHIFT` + :hwlabel:`SYNC` button                              Disables Sync mode.
+5         :hwlabel:`CUE` button                                                  Specifies, plays or recalls temporary cue point.
+5         :hwlabel:`SHIFT` + :hwlabel:`CUE` button                               Returns to the beginning of the song.
+6         :hwlabel:`PLAY/PAUSE` button                                           Plays or pause the song.
+6         :hwlabel:`SHIFT` + :hwlabel:`PLAY/PAUSE` button                        Hold to play backwards.
+7         Pad Mode Select                                                        See :ref:`Performance Pads <roland-dj-505-performancepads>`.
+8         Performance Pads                                                       See :ref:`Performance Pads <roland-dj-505-performancepads>`.
+9         :hwlabel:`PARAMETER` area                                              See :ref:`Performance Pads <roland-dj-505-performancepads>`.
+10        :hwlabel:`LOOP` area: :hwlabel:`AUTO  LOOP` button                     Set a beatloop at the current play position.
+10        :hwlabel:`LOOP` area: :hwlabel:`SHIFT` + :hwlabel:`AUTO  LOOP` button  Toggle looping.
```

Comment:
Is this mapped to `reloop_toggle`?
'Toggle looping' sounds a bit like the entire looping feature is toggled.
Would 'disable current loop or activate an existing loop' fit better?

Reply:
In 2.3 this is indeed mapped to `reloop_toggle` because `loop_enabled` is read-only. In 2.4 it toggles `loop_enabled`.

Reply:
Ok.